views/shared/_leadform.ejs

```diff
@@ -0,0 +1,178 @@
+    <div class="demo-modal__logo"><img src="/images/groups/deloitte.svg" /></div>
+    <div class="demo-modal__logo"><img src="/images/groups/nielsen.svg" /></div>
+    <div class="demo-modal__logo"><img src="/images/groups/paypal.svg" /></div>
+    <div class="demo-modal__logo"><img src="/images/groups/intel.svg" /></div>
+    <div class="demo-modal__logo"><img src="/images/groups/airbnb.svg" /></div>
+  </div>
+</div>
+
+<style>
+  .demo-modal {
+    padding: 0;
+    text-align: center;
+    max-width: 600px;
+    width: 90%;
+  }
+
+  .demo-modal__header {
+    background-image: linear-gradient(268deg, #405f8f, #1c3050);
+    padding: 40px 28px 14px;
+    border-radius: 5px 5px 0 0;
+  }
+
+  .demo-modal a {
+    color: #fff;
+    border-color: white;
+    font-weight: 600;
+  }
+
+  .demo-modal h2 {
+    color: #fff;
+    font-size: 32px;
+    font-weight: 700;
+    margin: auto;
+  }
+
+  .demo-modal p {
+    color: #fff;
+    font-size: 16px;
+    margin: 10px 0 24px;
+    line-height: 1.6;
+  }
+
+  .demo-modal__logos {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    padding: 0 10px 10px;
+    background: #FAFAFA;
+    border-radius: 0 0 10px 10px;
+  }
+
+  .demo-modal__logo {
+    margin: 5px 20px;
+    display: flex;
+    align-items: center;
+    height: 60px;
+  }
+
+  .demo-modal__logos img {
+    max-width: 100%;
+  }
+
+  .demo-modal__form-group label {
+    text-align: left;
+    margin
+  }
+
+  .demo-modal__form {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: space-evenly;
+    margin: 10px 0 20px;
+  }
+
+  .demo-modal__form-group {
+    width: 45%;
+  }
+
+  .demo-modal__form-group label {
+    display: block;
+    text-align: left;
+    margin: 12px 0 2px;
+    font-weight: 400;
+    color: #888;
+    font-size: 13px;
+  }
+
+  .demo-form__cta {
+    background-color: #ffc844;
+    border-color: #ffc844;
+    color: #3d4251;
+    margin-top: 20px;
+    font-weight: 700;
+    padding-bottom: 14px;
+    transition: background-color .3s cubic-bezier(.77,0,.175,1),border-color .3s cubic-bezier(.77,0,.175,1),color .3s cubic-bezier(.77,0,.175,1);
+  }
+
+  .demo-form__cta:hover {
+    background-color: #fbe28d;
+    border-color: #fbe28d;
+    color: #3d4251;
+  }
+
+  .demo-form__success {
+    text-align: center;
+    padding: 40px 0 30px;
+  }
+
+  .demo-form__success p {
+    color: #333;
+    margin: 0;
+  }
+
+  @media screen and (max-width: 600px) {
+    .demo-modal__form-group {
+      width: 90%;
+    }
+  }
+</style>
+
+<script>
+  $jq('.js-demo-form').on('submit', function(e) {
+    e.preventDefault();
+    $jq.ajax({
+      method: "POST",
+      url: "https://hooks.zapier.com/hooks/catch/2383820/wv86oo/",
+      data: $('.js-demo-form').serializeArray()
+    }).done(function(msg) {
+      $('.js-demo-form').html("<div class='demo-form__success'><h3>Thanks!</h3><p>We'll be in touch.</p></div>");
```

Comment:
Didn't see this feedback when testing locally, I just got redirected to the homepage with the form fields filled in as query variables in the URL